go/bundle/README.md

```diff
@@ -8,7 +8,11 @@ We currently provide three command-line tools: `gen-bundle`, `sign-bundle` and `
 
 `gen-bundle` command is a bundle generator tool. `gen-bundle` consumes a set of http exchanges (currently in the form of [HAR format](https://w3c.github.io/web-performance/specs/HAR/Overview.html), URL list file, or static files in a local directory), and emits a web bundle.
 
-`sign-bundle` command attaches a signature to a bundle. `sign-bundle` takes an existing bundle file, a certificate and a private key, and emits a new bundle file with cryptographic signature for the bundled resources added.
+`sign-bundle` command attaches a signature to a bundle. There are two supported ways to sign: using signed exchanges or integrity block, which can be separated using `-signType` `signedexchange` or `integrityblock` flag values. Without the signType flag, the default is `signedexchange`. 
```

Comment:
* This is an overview section so let's keep it simple. Can you move the explanation about `-signType` option to between `### sign-bundle` and `#### Using Signed Exchanges` ?
* The first option, which creates a [signatures section](https://github.com/WICG/webpackage/blob/main/extensions/signatures-section.md), is not really "signed exchanges", though it shares some concepts (e.g. certificate format) with [Signed Exchanges](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html). To avoid confusion, we should give it a different name. Maybe `-signType signaturessection`?